enterprise/app/codesearch/codesearch.tsx

```diff
@@ -87,13 +87,19 @@ export default class CodeSearchComponent extends React.Component<Props, State> {
               To see results, try entering a query. Here are some examples:
               <ul>
                 <li>
-                  <code className="inline-code">case:yes Hello World</code>
+                  <a href={"/search/?q=" + encodeUriComponent("case:yes Hello World")}>
+                    <code className="inline-code">case:yes Hello World</code>
+                  </a>
                 </li>
                 <li>
-                  <code className="inline-code">lang:css padding-(left|right)</code>
+                  <a href={"/search/?q=" + encodeUriComponent("lang:css padding-(left|right)")}>
+                    <code className="inline-code">lang:css padding-(left|right)</code>
+                  </a>
                 </li>
                 <li>
-                  <code className="inline-code">lang:go flag.String</code>
+                  <a href={"/search/?q=" + encodeUriComponent("lang:go flag.String")}>
```

Comment:
oops - these should be `encodeURIComponent`

Reply:
Done.